### DISCUSSION TITLE
Switch -> same value for multiple case

### DISCUSSION THREAD

**Discussion by rohitash-nagpal** [mockoon/mockoon#677] [Help] [answered]
How to handle switch in which we have same value/responses for multiple cases? please help on this.

**Comment by rohitash-nagpal** [comment, marked as answer]
@255kb -> can you please help on this? stuck at one place

**Reply by 255kb** [reply]
Coud you please post an example of what you tried until now? 
Thank you

**Reply by rohitash-nagpal** [reply]
Hi, I am expecting some functionality similar to this:

switch ($i) {
    case A:
    case B:
    case C:
        $letter = 'first';
        break;
    case D:
        $letter = 'second';
        break;
    default:
        $letter = 'third';
}

**Reply by rohitash-nagpal** [reply]
I want to aggregate these two cases in to a single one

{{#switch (body (concat 'catalog_ids.' @index))}}
           {{#case '146610051'}}
            "catalog_id": {{body (concat 'catalog_ids.' @index)}},
            "collection_ids": [
               655
            ]
        }
          {{/case}}
          {{#case '146610052'}}
            "catalog_id": {{body (concat 'catalog_ids.' @index)}},
            "collection_ids": [
               655
            ]
        }
        {{/case}}
         {{/switch}}

**Reply by 255kb** [reply]
You cannot achieve directly with the switch helper, however, you will be able to do this using Handlebars' partials. Here is a simplified case with an inline partial, receiving a parameter: 

```
{{#*inline "myPartial"}}
  "catalog_id": {{id}},
  "collection_ids": [
    655
  ]
{{/inline}}

{
  {{#switch (body 'test')}}
    {{#case '146610051'}}
      {{> myPartial id='146610051'}}
    {{/case}}
    {{#case '146610052'}}
      {{> myPartial id='146610052'}}
    {{/case}}
  {{/switch}}
}
```

**Reply by rohitash-nagpal** [reply]
works fine, thank you so much for your swift response :)

**Reply by rohitash-nagpal** [reply]
has this been not added to documentation yet, couldn't find it

**Reply by 255kb** [reply]
no it's not, because iit's part of Handlebars documentation. What is document are Mockoon's own helpers.
